grails-app/i18n/messages_es.properties

```diff
@@ -482,7 +482,7 @@ default.notes.label=Notes
 default.null.message=La propiedad [{0}] de la clase [{1}] no puede ser nulo
 #default.number.format                 = 0
 default.numItems.label=\# items
-default.number.format=\# \# \#, \# \# \#, \# \# \#0.00
+default.number.format=\# \# \#, \# \# \#, \# \# \#
```

Comment:
@drodzewicz I think this one is wrong. Am I right?

Reply:
yes, it looks like it's rolling back the change that I have made in #3954